### PR TITLE
#69: Adjustable list width

### PR DESCRIPTION
**Pull request by mattb555**
Because of how I shifted the CSS burden of the grid layout, the covers are now a lot bigger than they used to be, I'm ok with that because the lists stay generally the same size when shifting view, but some of the cover images aren't really the right resolution to be blown up that much.

### CODE REVIEW

File: src/components/Lists/ListSettings.vue [hunk 1]
```diff
@@ -56,6 +56,23 @@
             </button>
         </section>
 
+        <section v-if="hasMultipleGames">
+            <h4>List Width</h4>
```
Comment (Swinkid) [inline src/components/Lists/ListSettings.vue]: Can you add this as a translation string? Recently finished adding all hard coded strings to i18n

Comment (Swinkid) [reply src/components/Lists/ListSettings.vue]: See #107 and #108

Comment (mattb555) [reply src/components/Lists/ListSettings.vue]: Sure thing, I'm on my way out the door right now so I won't have it done for a while but it looks like there are actually a number of hard coded strings still on the page that I'll whack.

Comment (Swinkid) [reply src/components/Lists/ListSettings.vue]: Cheers buddy. I thought I caught them all, might be worth merging the latest master into your branch.

**Comment by mattb555** [comment]
Removed hard coded Strings in ListSettings, put TODO's in non English language files to note that translations required for said Strings.

**Pull request merged**